app/controllers/decidim/participatory_documents/admin/documents_controller.rb

```diff
@@ -74,6 +79,10 @@ def add_snippets
         def document
           @document ||= Decidim::ParticipatoryDocuments::Document.find_by(component: current_component)
         end
+
+        def annotations
```

Comment:
```suggestion
        def zones
```

The entity holding the participation will be a "zone", which is a group of annotations (or box). Let's change that.